<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExibeFoto</class>
 <widget class="QMainWindow" name="ExibeFoto">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detalhes da Imagem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="GraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>611</width>
          <height>651</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>668</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Fechar</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/resourcesIcon.qrc">
            <normaloff>:/IconesMenu/error.png</normaloff>:/IconesMenu/error.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>324</width>
          <height>651</height>
         </size>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tabExifTree">
         <attribute name="icon">
          <iconset resource="../resources/resourcesIcon.qrc">
           <normaloff>:/IconesMenu/exif.png</normaloff>:/IconesMenu/exif.png</iconset>
         </attribute>
         <attribute name="title">
          <string notr="true">EXIF</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treeWidgetExif">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="indentation">
             <number>10</number>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>false</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="headerHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="headerMinimumSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">Data</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Valor</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabIptcTree">
         <attribute name="icon">
          <iconset resource="../resources/resourcesIcon.qrc">
           <normaloff>:/IconesMenu/iptc_top_logo.png</normaloff>:/IconesMenu/iptc_top_logo.png</iconset>
         </attribute>
         <attribute name="title">
          <string notr="true">IPTC</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treeWidgetIptc">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="uniformRowHeights">
             <bool>false</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="headerMinimumSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">Data</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Valor</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabXmpTree">
         <attribute name="icon">
          <iconset resource="../resources/resourcesIcon.qrc">
           <normaloff>:/IconesMenu/adobe-xmp.png</normaloff>:/IconesMenu/adobe-xmp.png</iconset>
         </attribute>
         <attribute name="title">
          <string notr="true">XMP</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treeWidgetXmp">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="headerMinimumSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">Data</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Valor</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionSalvarComo"/>
    <addaction name="separator"/>
    <addaction name="actionFechar"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionManipularExifIptc"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuEditar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFechar">
   <property name="text">
    <string>Fechar</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar </string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSalvarComo">
   <property name="text">
    <string>Salvar Como...</string>
   </property>
  </action>
  <action name="actionManipularExifIptc">
   <property name="text">
    <string>Manipular Exif/Iptc</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>./Forms/ExibeFoto/GraphicsView.h</header>
   <slots>
    <slot>formEditMetadatas()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resourcesIcon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFechar</sender>
   <signal>triggered()</signal>
   <receiver>ExibeFoto</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionManipularExifIptc</sender>
   <signal>triggered()</signal>
   <receiver>ExibeFoto</receiver>
   <slot>formEditMetadatas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExibeFoto</receiver>
   <slot>fechar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>841</x>
     <y>710</y>
    </hint>
    <hint type="destinationlabel">
     <x>773</x>
     <y>717</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExibeFoto</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>826</x>
     <y>697</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>708</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>formEditMetadatas()</slot>
  <slot>fechar()</slot>
 </slots>
</ui>
